File: frameworks/C#/aspnet-stripped/src/App_Code/JsonHandler.cs
using System.Web;
using System.Web.Script.Serialization;

public class JsonHandler : IHttpHandler
{
    bool IHttpHandler.IsReusable
    {
        get { return true; }
    }

    void IHttpHandler.ProcessRequest(HttpContext context)
    {
        HttpResponse response = context.Response;
        response.ContentType = "application/json";
        response.Write(new JavaScriptSerializer().Serialize(new { message = "Hello, World!" }));
    }
}

File: frameworks/Java/jetty/src/main/java/hello/JsonHandler.java
package hello;

import java.io.IOException;
import java.util.Collections;
import java.util.Map;

import javax.servlet.ServletException;
import javax.servlet.http.HttpServletRequest;
import javax.servlet.http.HttpServletResponse;

import org.eclipse.jetty.http.HttpField;
import org.eclipse.jetty.http.HttpHeader;
import org.eclipse.jetty.http.PreEncodedHttpField;
import org.eclipse.jetty.server.Request;
import org.eclipse.jetty.server.handler.AbstractHandler;
import org.eclipse.jetty.util.ajax.JSON;



public class JsonHandler extends AbstractHandler
{
    private JSON json = new JSON();
    HttpField contentType = new PreEncodedHttpField(HttpHeader.CONTENT_TYPE,"application/json");

    @Override
    public void handle(String target, Request baseRequest, HttpServletRequest request, HttpServletResponse response) throws IOException, ServletException
    {
        baseRequest.setHandled(true);
        baseRequest.getResponse().getHttpFields().add(contentType);  
        Map<String,String> map = Collections.singletonMap("message","Hello, World!");        
        json.append(response.getWriter(),map);
    }

}
